src/mjolnir/elevationbuilder.cc

```diff
@@ -315,11 +315,16 @@ namespace mjolnir {
 
 void ElevationBuilder::Build(const boost::property_tree::ptree& pt,
                              std::deque<baldr::GraphId> tile_ids) {
-  auto elevation = pt.get_optional<std::string>("additional_data.elevation");
+  boost::optional<std::string> elevation = pt.get_optional<std::string>("additional_data.elevation");
   if (!elevation || !filesystem::exists(*elevation)) {
     LOG_WARN("Elevation storage directory does not exist");
     return;
   }
+  boost::optional<bool> addElevation = pt.get_optional<bool>("mjolnir.store_elevation_along_edges");
+  if (addElevation == false) {
+    LOG_WARN("Elevation won't be stored in tiles");
+    return;
```

Comment:
so by returning here you disable BOTH the weighted_grade as well as storing elevation along the edges. the former doesnt cost any extra size to the data and can vastly improve certain types of mobility (pedestrian and bike, electric once we add that) whereas the latter will cost size. i assume you wanted this to avoid the extra size in the tiles in which case we should differentiate between the two actions here. if the intent was just to disable elevation data use generally then you can simply poin the directory for elevation at /dev/null. let me know what your intent is here

Reply:
Also, when I first added this I believe elevation for roads in Valhalla tiles added about 1-2% to the size of the data. That is why I didn't think it was necessary to add a config option.